src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -143,24 +154,47 @@ internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount
             }
         }
 
+        internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount, IEnumerable<string> languages)
+        {
+            foreach (string language in languages)
+            {
```

Comment:
Is everything in this `for` loop identical to `InitializeJobhostLanguageWorkerChannelAsync(int attemptCount, string language)`? Can we just call that inside this loop to ensure the logic is the same for both cases? 

something like...
```
foreach (string language in languages)
{
    await InitializeJobhostLanguageWorkerChannelAsync(0, language)
}
```

Reply:
If that's the case, we can probably move that foreach up to line 137 to avoid an additional method overload